task/build-vm-image/0.1/build-vm-image.yaml

```diff
@@ -65,7 +65,7 @@ spec:
         name: varlibcontainers
   steps:
     - name: use-trusted-artifact
-      image: quay.io/redhat-appstudio/build-trusted-artifacts:latest@sha256:c3d440309b1a24cdfa8a2c89b668ca56952185f5c50e7abfa14b2aaafd5e2846
+      image: quay.io/redhat-appstudio/build-trusted-artifacts:latest@sha256:0379f5c2feec921a01300b386ce9414b10acfbc4499b187700d972b2360e1585
```

Comment:
@Kousalya1998 it looks like your rebase went awry. Please remove these unrelated changes.

Reply:
squashed it

Reply:
when I rebased it again I dont see those unrelated changes now.